org.dupot.easyflatpak.yml

```diff
@@ -0,0 +1,35 @@
+app-id: org.dupot.easyflatpak
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: dupot_easy_flatpak
+separate-locales: false
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=ipc
+  - --device=dri
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
Why is flatpak-spawn required? This is not allowed by default.

Reply:
my application call flatpak to override permission for some application after their installation 
for example for steam to add game's user folder

Reply:
You said in the description `Application to simplify installation of some flatpak` now you are saying it needs to override permissions.

What exactly is this app?

Reply:
i just updated the readme..
The idea is very simple: you have a folder which store recipies for each application in json format.
In each json file, you have title,description, flatpak id + a list of override action needed after installation
For example, for steam, after installation, the application ask used to setup the folder where his games are stored to add it with --filesystem override

Reply:
can you open a PR for the exception? https://docs.flathub.org/docs/for-app-authors/linter#exceptions

This is the error code https://docs.flathub.org/docs/for-app-authors/linter#finish-args-flatpak-spawn-access

Reply:
Thanks for advice
i did it : https://github.com/imikado/flatpak-builder-lint/pull/1
Should i click on "merge pull request" button ?

Reply:
i clicked on merge pull request

Should i do other modifications ?

Reply:
Don't merge, click on Open pull request. https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request#creating-the-pull-request